charts/s3gw/templates/ingress-traefik.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if or .Values.ingress.nginx.enabled .Values.ingress.traefik.enabled }}
```

Comment:
same considerations from above, the logic can be univocal assuming only Traefik.